INSTALL

```diff
@@ -508,6 +508,11 @@ MFEM_USE_MKL_CPARDISO = YES/NO
    MFEM_USE_LAPACK=YES, verify that the MKL LAPACK libraries are used. The
    OpenMP capabilities are disabled at link time.
 
+MFEM_USE_MOONOOLITH = YES/NO
+   Enables the interface ParMoonolith for parallel non-conforming, non-matching,
+   variational, volumetric mesh information transfer using ParMoonolith. It
+   requires MFEM_USE_MPI=YES, MFEM_USE_METIS=YES, and MOONOLITH_DIR=<path to install>
```

Comment:
It appears that you have implemented a serial `MortarAssembler` and examples that are serial. Are `MFEM_USE_MPI=YES` and metis strictly required for those?

Reply:
Fixed it. Now it supports both MFEM_USE_MPI=YES and MFEM_USE_MPI=NO without any particular requirement.